zokrates_core/src/proof_system/mod.rs

```diff
@@ -34,5 +34,7 @@ pub trait ProofSystem {
         proving_key: Vec<u8>,
     ) -> String;
 
-    fn export_solidity_verifier(&self, vk: String, is_abiv2: bool) -> String;
+    fn export_solidity_verifier(&self, vk: String, abi_v2: bool) -> String;
```

Comment:
I think `is_` is a good prefix for boolean variables. The Rust way would actually be to define 
```
enum Abi {
   V1,
   V2
}
```

Maybe we should consider that?

Reply:
Introducing enum is the way to go here imo